docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM debian:bullseye-slim
+
+ARG DIRECTORY
+ARG BINARY
+RUN echo "Copying $DIRECTORY/$BINARY into container"
```

Comment:
nit: prefer wrapping variables inside `{}` , like: `${PATH}` (otherwise it would break on a space).